src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -822,15 +822,6 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
 
         dduf_entries = None
         if dduf_file:
-            if not is_huggingface_hub_version(">", "0.26.3"):
```

Comment:
Why are we removing this?

Reply:
Minimum version changed `huggingface-hub>=0.27.0`

Reply:
With the dependency version updated we don't need to check the version to import `read_dduf_file`, but we could keep the same dependency version and change the version here instead.

Reply:
Ah yeah got it.